gensim/models/nmf_pgd.pyx

```diff
@@ -8,26 +8,23 @@
 
 from libc.math cimport sqrt, fabs, copysign
 from cython.parallel import prange
+import numpy as np
+cimport numpy as np
 
 cdef double fmin(double x, double y) nogil:
```

Comment:
don't think you need this. cython has native support  of min/max. check it.
and as another option you can add `inline`  in function definition.

Reply:
No, this is a mandatory piece: you'll be surprised, but windows compiler for py27 is very old and wired, have issues even with this simple thing, for this reason, we have it here.

Reply:
=(, so just add inline

Reply:
ping @anotherbugmaster